src/yarpmanager/src-builder/arrow.cpp

```diff
@@ -57,35 +57,29 @@ const qreal Pi = 3.14;
 
 
 
-Arrow::Arrow(BuilderItem *startItem, BuilderItem *endItem,int id, Manager *safeManager,
-             bool isInApp,bool editingMode, BuilderItem *parent)
-    : BuilderItem(parent),textLbl("",this)
+Arrow::Arrow(BuilderItem *startItem, BuilderItem *endItem,int _id, Manager *safeManager,
```

Comment:
I didn't notice this kind of changes, and I merged it already, but there is no need to rename the `id` variable, you can safely initialize it as `id(id)`

Reply:
Yes I know but I saw somewhere in yarp the `id(_id)` style and it seemed more clear to me so I adopted it. If you prefer the other one I will keep in mind.